notebooks/05_operationalize/als_movie_o16n.ipynb

```diff
@@ -92,8 +92,11 @@
     "from azureml.core.model import Model\n",
```

Comment:
<p>just realized that this is repeated in the first cell <code>from azureml.core.compute import AksCompute, ComputeTarget</code>, we can erase it from here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/964/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>